frontend/package.json

```diff
@@ -84,7 +84,7 @@
     "react": "^18",
     "react-chartjs-2": "^2.9.0",
     "react-datepicker": "^2.14.1",
-    "react-dom": "^17",
+    "react-dom": "^18",
```

Comment:
I think we need to fix the sizing of the legend paper:
![Screenshot 2024-07-05 at 11 34 47 AM](https://github.com/WFP-VAM/prism-app/assets/16843267/c04cad32-cab4-4766-85be-ee15a03a712a)